### PR TITLE
Update serving and kourier to reference 0.19 images

### PR DESCRIPTION
**Pull request by nak3**
This patch changes to:
- Update serving 0.19.0 & kourier 0.19.1 in `olm-catalog/serverless-operator/project.yaml`.
- Remove `001-liveness.patch`.
- Add webhook pdb patch to `003-serving-pdb.patch`.
- Add domainmapping resource yaml.

/cc @markusthoemmes @matzew

### CODE REVIEW

**Comment by nak3** [comment]
/retest

TestServicePostUpgrade failed due to insufficient cpu:

```
failed with message: 0/6 nodes are available: 3 Insufficient cpu, 3 node(s) had taint {node-role.kubernetes.io/master:
```

File: olm-catalog/serverless-operator/project.yaml [hunk 1]
```diff
@@ -19,9 +19,9 @@ requirements:
...
 
 dependencies:
-  serving: 0.18.2
+  serving: 0.19.0
   eventing: 0.18.6
   eventing_contrib: 0.18.0
-  kourier: 0.18.1
+  kourier: 0.19.1
   cli: 0.18.4
   maistra: 1.1.5
```
Comment (markusthoemmes) [inline olm-catalog/serverless-operator/project.yaml:27]: Let's try bump this to `2.0.0` too, should work now.

**Review by markusthoemmes — APPROVED** [review]
This looks exactly correct, thank you!

I'll keep the LGTM off until we have the upgrade tests sorted.

/hold

**Comment by nak3** [comment]
Again, lack of resources. domain-mapping webhook and controller added 30m + 100m CPU requests though..

```
failed with message: 0/6 nodes are available: 3 Insufficient cpu, 3 node(s) had taint {node-role.kubernetes.io/master: }, that the pod didn't tolerate.
```

**Comment by nak3** [comment]
Not only upgrade test but also upstream test is broken now. 
Activator's endpoint is missing. Same error happens on https://github.com/openshift-knative/serverless-operator/pull/753.

```
    e2e.go:169: Did not see activator endpoints in public service for g-r-p-c-unary-ping-v-bgddaywm-xw476.Last received values: Activator:[]PubSvc: [], WantActivators 0
    grpc_test.go:381: Never got Activator endpoints in the service: timed out waiting for the condition
```

**Comment by nak3** [comment]
/retest

Infra issue. Only `4.6-operator-e2e-aws-ocp-46` has a strange issue but let's see another try.

**Comment by nak3** [comment]
All image digests are not resolved well. I will retest later.

```
failed to resolve image to digest: Head \"https://registry.svc.ci.openshift.org/v2/openshift/knative-v0.19.0/manifests/knative-serving-test-pizzaplanetv1\": context deadline exceeded."}
```

**Comment by matzew** [comment]
@nak3 Eventing bits got merged, I think this needs a new rebase ... :-(

**Comment by nak3** [comment]
/test 4.6-upstream-e2e-aws-ocp-46

infra issue.

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Pull request merged**